### PR TITLE
Statistics

### PR DESCRIPTION
**Pull request by sanemorak**
- calculation of mean Acceleration Magnitude and Gyroscope Magnitude
- calculation of over-threshold AccelMag and GyroMag means
- applying Pearson's r, Speraman's Rank and Kendall's Rank Correlation tests for both overall means and over-threshold means
- Student's t-test for female/male groups

### CODE REVIEW

File: imu.ipynb [hunk 1]
```diff
@@ -209,7 +511,7 @@
     "    acc_3.xaxis.set_major_formatter(mdates.DateFormatter('%H:%M'))\n",
     "    acc_3.set_xticklabels(acc_3.get_xticklabels(), rotation=45, horizontalalignment='right')\n",
     "\n",
-    "    fname = 'results/accel-'+key+'.png'\n",
+    "    fname = 'results/accel'+key+'.png'\n",
```
Comment (vayvy) [inline imu.ipynb:514]: Here the "-" is missing after the term "accel".

File: imu.ipynb [hunk 2]
```diff
@@ -237,7 +562,7 @@
     "    acc_mag.xaxis.set_major_formatter(mdates.DateFormatter('%H:%M'))\n",
     "    acc_mag.set_xticklabels(acc_mag.get_xticklabels(), rotation=45, horizontalalignment='right')\n",
     "\n",
-    "    fname = 'results/accelmag-'+key+'.png'\n",
+    "    fname = '/Users/sanemcansinorak/Statistics/data/accelmag'+key+'.png'\n",
```
Comment (vayvy) [inline imu.ipynb:565]: Here you forgot to change the directory back to the original form.

File: imu.ipynb (hunk 1 above)
Comment (vayvy) [inline imu.ipynb:514]: Here the "-" is missing after the term "accel".

File: imu.ipynb (hunk 2 above)
Comment (vayvy) [inline imu.ipynb:565]: Here you forgot the change the path back to its original form.

File: imu.ipynb [hunk 3]
```diff
@@ -246,12 +571,24 @@
...
+      "\u001b[0;31mNameError\u001b[0m: name 'gyro' is not defined"
+     ]
+    }
+   ],
    "source": [
     "# Plotting axes of gyroscope seperatly\n",
     "for key in imu_data:\n",
     "    data = imu_data[key].loc[:,('Timestamp','Gyro X', 'Gyro Y', 'Gyro Z')]\n",
-    "    gyro = gyro.plot(x = 'Timestamp', linewidth= lw)\n",
+    "    gyro = sns.lineplot(x = 'Timestamp', linewidth= lw)\n",
```
Comment (vayvy) [inline imu.ipynb:591]: Why did you change the function from plot() to linepolt()? Does it give better results?

File: imu.ipynb [hunk 4]
```diff
@@ -266,7 +603,7 @@
     "    gyro.xaxis.set_major_formatter(mdates.DateFormatter('%H:%M'))\n",
     "    gyro.set_xticklabels(gyro.get_xticklabels(), rotation=45, horizontalalignment='right')\n",
     "\n",
-    "    fname = 'results/gyro-'+key+'.png'\n",
+    "    fname = 'results/gyro'+key+'.png'\n",
```
Comment (vayvy) [inline imu.ipynb:606]: Here the "-" is missing after the term "gyro".

File: imu.ipynb [hunk 5]
```diff
@@ -283,7 +620,7 @@
     "\n",
     "    # Set plot title\n",
     "    title = 'Magnitude of Angular Velocity' + ' (' + key[:5] + ', ' + 'Day ' + key[6] + ')'\n",
-    "    gyro_mag.set_title(, fontdict={'size': title_size})\n",
+    "    gyro_mag.set_title(title, fontdict={'size': title_size})\n",
```
Comment (vayvy) [inline imu.ipynb:623]: Thank you ❤️

File: imu.ipynb [hunk 6]
```diff
@@ -294,7 +631,7 @@
     "    gyro_mag.xaxis.set_major_formatter(mdates.DateFormatter('%H:%M'))\n",
     "    gyro_mag.set_xticklabels(gyro_mag.get_xticklabels(), rotation=45, horizontalalignment='right')\n",
     "\n",
-    "    fname = 'results/gyromag-'+key+'.png'\n",
+    "    fname = '/Users/sanemcansinorak/Statistics/data/gyro-mag'+key+'.png'\n",
```
Comment (vayvy) [inline imu.ipynb:634]: Here you forgot the change the path back to its original form.

File: questionnaire.ipynb
Comment (vayvy) [inline questionnaire.ipynb:1]: This file seems fine. The only difference is the outputs. I think before committing changes to our branches we should try to always clear all the outputs to mitigate this overhead.

**Review by vayvy — CHANGES_REQUESTED** [review]

File: imu.ipynb (hunk 3 above)
Comment (sanemorak) [reply imu.ipynb]: No reason, I think it gave an error at the beginning. Reverted it back

File: imu.ipynb (hunk 5 above)
Comment (sanemorak) [reply imu.ipynb:623]: ❤️

**Review by vayvy — APPROVED** [review]
Looks good. I'll add the changes to the main branch. I will make a new pull request in a bit to fix a few minor fixes to the formatting of the script :)

**Pull request merged**